src/components/Gradebook/StatusAlerts.test.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { StatusAlerts, mapDispatchToProps, mapStateToProps } from './StatusAlerts';
+import { closeBanner } from '../../data/actions/grades';
+
+describe('StatusAlerts', () => {
+  const renderStatusAlerts = (params) => {
+    const alertParams = {
+      handleCloseSuccessBanner: jest.fn().mockName('handleCloseSuccessBanner'),
+      showSuccessBanner: false,
+      ...params,
+    };
+    const alerts = renderer.create(<StatusAlerts {...alertParams} />);
```

Comment:
FWIW, I much much prefer enzyme.shallow over react-test-renderer.  Does not try to interpret the children out, which makes for cleaner snapshots.  (also supports mock-naming better, but that isn't an issue for this component)